CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - observability: extend response, and request payload size histogram buckets up to 256MB.
 ### Fixed
 - introspection: wrong display of proto procedure name.
+- error details: propagation of protobuf error details did not work if the any message was not
```

Comment:
This does not say what you fixed but what the issue is. Can you rephrase? Something like
`error details: correct propagation of protobuf error details not in gogo proto registry`